CONTRIBUTING.md

```diff
@@ -37,6 +37,18 @@ Submit a pull request
 - Please follow our code style. You can use Checkstyle within Android Studio using our [coniguration file](https://github.com/AntennaPod/AntennaPod/blob/develop/config/checkstyle/checkstyle-new-code.xml).
 - Please only change the English string resources. Translations are handled on [Transifex](https://www.transifex.com/antennapod/antennapod/).
 
+Building From Source
+--------------------------
+1. Clone this repository
+
+(For example:)
+`git clone https://github.com/AntennaPod/AntennaPod.git`
```

Comment:
I think users should clone their own forks, so they don't have to mess with changing the remotes when they want to open a PR later.